index.html

```diff
@@ -87,28 +94,32 @@ <h2 class="portfolio-title">Tonic</h2>
                     <li>JAVASCRIPT</li>
                 </ul>
                 <button class="see-demo-btn" type="button">See Project</button>
+                </div>
             </div>
         </div>
     </section>
     <section class="aboutme">
-        <h2 class="title">I’m Mudasir
-            Welcome to my Site!</h2>
-        <p class="description">I’m a fullstack web developer! I can help you build a product website, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <h3><a href="#" class="action-text">LET’S CONNECT</a></h3>
-        <ul class="social-media">
-            <li><a href="https://twitter.com/mudasirissah"><i class="fa-brands fa-twitter"></i></a></li>
-            <li><a href="https://linkedin.com/in/mudasirissah"><i class="fa-brands fa-linkedin"></i></a></li>
-            <li><a href="https://github.com/mudasiri"><i class="fa-brands fa-github"></i></a></li>
-            <li><a href="https://github.com/mudasiri"><i class="fa-brands fa-medium"></i></a></li>
-        </ul>
-        <button class="see-demo-btn">Get my resume</button>
+        <div>
+            <h2 class="title">About <br>
+                Myself</h2>
+            <p class="description">I’m a fullstack web developer! I can help you build a product website, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h3><a href="#" class="action-text">LET’S CONNECT</a></h3>
+            <ul class="social-media">
+                <li><a href="https://twitter.com/mudasirissah"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="https://linkedin.com/in/mudasirissah"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="https://github.com/mudasiri"><i class="fa-brands fa-github"></i></a></li>
+                <li><a href="https://github.com/mudasiri"><i class="fa-brands fa-medium"></i></a></li>
+            </ul>
+            <button type="button" class="see-demo-btn">Get my resume</button>
+        </div>
 
         <ul class="skils-ul">
             <li>
                 <div class="flex-skill-title">
                   <h2>Languages</h2>
                   <i class="fa-sharp fa-solid fa-angle-down"></i>
                 </div>
+           <div>
             <ul class="sub-languages">
```

Comment:
- [ ] Kindly ensure that you align the languages correctly in a horizontal direction for the desktop view as it is in the Figma design

**This is how I am seeing your project so far:**

![image](https://user-images.githubusercontent.com/86318284/192342732-f629523a-670b-48cb-a678-3ea1fd52f12b.png)


**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/192340596-21303e7c-bf08-4487-9017-8401edf94f6b.png)